spec/openapi.yaml

```diff
@@ -7711,6 +7711,309 @@ paths:
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+  /jobs-api/v3/projects/{projectId}/custom-fields:
+    get:
+      summary: Find custom fields for specified project
+      description: >-
+        Get all custom fields for specified project.
+      tags:
+        - Jobs
+      operationId: getProjectCustomFields
+
+      parameters:
+        - description: A unique identifier for a project.
+          in: path
+          name: projectId
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CustomFields'
+              examples:
+                response:
+                  value:
+                    response:
+                      code: SUCCESS
+                      data:
+                        items:
+                          - fieldUid: ngahq6dgh4ed
+                            type: SELECTBOX
+                            fieldName: field-name
+                            enabled: true
+                            required: true
+                            searchable: true
+                            displayToTranslators: true
+                            options:
+                              - option1
+                              - option2
+                            defaultValue: default-field-name
+                            description: custom field description
+                        totalCount: 1
+          description: OK
+        400:
+          $ref: '#/components/responses/Error400ResponseDefinition'
+        401:
+          $ref: '#/components/responses/Error401ResponseDefinition'
+        404:
+          description: Not found error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error404Response'
+              examples:
+                response:
+                  value:
+                    response:
+                      code: NOT_FOUND_ERROR
+                      errors:
+                        - key: null
+                          message: null
+                          details: null
+
+        429:
+          $ref: '#/components/responses/Error429ResponseDefinition'
+        500:
+          $ref: '#/components/responses/Error500ResponseDefinition'
+
+    post:
+      summary: Assign new custom fields to project
+      description: >-
+        Accepts new values for assignment. If empty list is passed - all assignments are deleted
```

Comment:
lets append `Order of the fields in the request is preserved during saving.` Also to be consistent make sure all descriptions are ended with period (full stop point).